types/artifacts.d.ts

```diff
@@ -283,6 +290,17 @@ declare global {
         map?: undefined;
       }
 
+      export interface Bundle {
```

Comment:
i was going to say we don't want the same data (ScriptElement) reported in two separate artifacts... but then realized Bundle is a computed artifact.

at some point we should separate the types of our computed artifacts from our gathered artifacts.   tbh kinda weird that they are all in the same spot.